src/main/java/uk/gov/hmcts/reform/notificationservice/service/NotificationService.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.notificationservice.service;
+
+import org.slf4j.Logger;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.notificationservice.data.Notification;
+import uk.gov.hmcts.reform.notificationservice.data.NotificationRepository;
+
+import java.util.List;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+@Service
+public class NotificationService {
+
+    private static final Logger log = getLogger(NotificationService.class);
+
+    private final NotificationRepository notificationRepository;
+
+    public NotificationService(
+        NotificationRepository notificationRepository
+    ) {
+        this.notificationRepository = notificationRepository;
+    }
+
+    public void processPendingNotifications() {
+        List<Notification> notifications = notificationRepository.findPending();
+
+        log.info("Notifications to process: {}", notifications.size());
```

Comment:
This class doesn't do it's job. Why not create a PR with a complete version instead?

Reply:
as description says:

> Doing quick, small one so to not overlap too much with other tickets

Reply:
I don't think this is a correct cut. It makes sense to add changes first in dependencies (can even be one at a time) and go up, but when we introduce a class that's responsible for a certain task, it should do it. Otherwise, its tests must be incorrect/incomplete and it's very hard to verify later if they're not missing. We've had cases in this kind of approach in the past, leaving us with missing tests.